### PR TITLE
CRD Support - Frontend

### PR DESCRIPTION
**Pull request by eloyekunle**
- [x] Add CRD ActionBar buttons
- [x] Add Status icon to CRD list
- [x] Paginate CRD Objects list
- [x] Extend Verber to support CRD Objects.
- [x] Add Edit & Delete ActionBar buttons.
- [x] Add Raw object data display.

Ref - #2493

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@eloyekunle Any progress here?

**Comment by eloyekunle** [comment]
@maciaszczykm Yes, I'll push updates today or tomorrow.

**Comment by eloyekunle** [comment]
Current Screenshots:

**1. CRD List Page**
localhost_8080_ (1)

**2. CRD Details Page**
localhost_8080_

**Comment by maciaszczykm** [comment]
What is missing? Can we review?

**Comment by eloyekunle** [comment]
@maciaszczykm 

I want to update the CRD Objects list to a paginated display, then it will be ready for review.

The CRD Object Detail Page will then come in via another PR.

**Comment by eloyekunle** [comment]
@maciaszczykm this is ready for a review.
/assign maciaszczykm jeefy

**Comment by eloyekunle** [comment]
Current Views

**1. CRD Detail View**
image

**2. CRD Object Detail**
Currently minimal. Separate issue at #4123 
image

**Comment by maciaszczykm** [comment]
Taking a look.

File: i18n/messages.xlf [hunk 1]
```diff
@@ -3789,6 +3975,7 @@
           <context context-type="linenumber">320</context>
         </context-group>
       </trans-unit>
+<<<<<<< HEAD
```
Comment (floreks) [inline i18n/messages.xlf]: This part looks like a merge conflict.

**Comment by floreks** [comment]
Overall code looks really good. I'll test how it works and if there are no issues we can merge once you fix merge conflict.

**Comment by maciaszczykm** [comment]
Besides issue in translation files everything else looks good. Good work! Fix and we merge.

/approve

File: i18n/messages.xlf (hunk 1 above)
Comment (eloyekunle) [reply i18n/messages.xlf]: I've fixed it.

Comment (eloyekunle) [reply i18n/messages.xlf]: Good catch! :+1:

**Pull request merged**